src/plugins/analysis/users_and_passwords/install.sh

```diff
@@ -8,24 +8,47 @@ echo "------------------------------------"
 
 sudo -EH pip3 install --upgrade git+https://github.com/fkie-cad/common_helper_passwords.git || exit 1
 
+mkdir -p bin/john/
 if [ "$1" = "fedora" ]
-then 
-	# installing JohnTheRipper
-	sudo dnf install -y john || exit 1
-
-	# link to path since fedora does not include /usr/sbin
-	sudo ln -sfn /usr/sbin/john /usr/local/bin || exit 1
+then
+  (
+    cd bin/john/
```

Comment:
changing the directory is still not necessary here and in line 23 (since nothing related to the directory happens here)